runtime/near-runtime-fees/src/lib.rs

```diff
@@ -10,6 +10,8 @@ pub type Gas = u64;
 
 /// Costs associated with an object that can only be sent over the network (and executed
 /// by the receiver).
+/// NOTE: `send_sir` or `send_not_sir` fees are usually burned when the item is being created.
```

Comment:
In the `genesis_config.json` these two fees are always equal.  Wouldn't you expect `send_sir` to be cheaper?

Reply:
All of them are handled the same right now. We don't have optimizations for `sir` and `not_sir`